zstacklib/zstacklib/utils/sanlock.py

```
@@ -1,20 +1,91 @@
 logger = log.get_logger(__name__)
 
+def is_sanlock_lease_corrupted(err):
+    return any(str(code) in err for code in SANLK_LEASE_CORRUPTED)
+
+
+def retry_if_sanlock_lease_corrupted(times=3, sleep_time=3):
+    def wrap(f):
+        @functools.wraps(f)
+        def inner(*args, **kwargs):
+            orig_except = None
+            for i in range(0, times):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)